### ISSUE TITLE
Neo4j 3.x

### ISSUE THREAD

**Issue by galpin** [niclashoyer/neo4j-sparql-extension#50]
Is this project dead for Neo4j 3.x?

Neo4j currently fails to load with the attached error.

Thanks,

[debug.txt](https://github.com/niclashoyer/neo4j-sparql-extension/files/384635/debug.txt)

**Comment by niclashoyer** [comment]
Yes, I'm sorry. This project depends heavily on https://github.com/tinkerpop/blueprints and that project is now a part of the Apache Software Foundation (including breaking changes). I did not yet have enough time to look into the new TinkerPop 3.0 and what changes are needed. It may also be possible that they don't support Neo4j 3.x at all.

**Comment by niclashoyer** [comment]
I had a quick look. It seems that TinkerPop 3.0 supports Neo4j 3.x, but the [Sesame Sail 2.x Ouplementation](https://github.com/tinkerpop/blueprints/tree/master/blueprints-sail-graph) is no longer available, so it is not possible to port this project to TinkerPop 3.0.

TinkerPop 3.0 supports SPARQL, but that implementation uses Jena ARQ instead of Sesame. That would require a full rewrite, because Jena ARQ is a completely different Java/RDF/SPARQL API.
